specs/linux_tpm_pcr_registry.md

```diff
@@ -28,6 +30,70 @@ In both cases it is important that data measured into the PCRs is carefully chos
    <th><strong>Log</strong></th>
    <th><strong>Use Reported By</strong></th>
   </tr>
+
+  <tr>
+   <td style="background-color:#fff3bf;"><p style="text-align: right"><strong>0</strong></p></td>
+   <td style="background-color:#AEA;"><code style="background-color:#AEA;">Firmware 💻</code></td>
```

Comment:
the background color seems wrong. should be grey, not green, like for the other firmware entries

Reply:
Search-and-replace hard. I'll submit a follow-up.